rviz_rendering/src/rviz_rendering/render_system.hpp

```diff
@@ -52,6 +49,11 @@
 #include <OgreOverlaySystem.h>
 #include <RenderSystems/GL/OgreGLPlugin.h>
 
+#include <cstdint>
+#include <string>
+
+#include <QDir> // NOLINT
```

Comment:
These should be included before the Ogre headers and outside of the warnings pragma. I would just move them back...